src/main/java/frc/robot/Controls.java

```diff
@@ -101,11 +101,20 @@ private void configureDrivebaseBindings() {
                   ChassisSpeeds targetSpeeds =
                       new ChassisSpeeds(
                           -driverController.getLeftY()
-                              * MaxSpeed, // Drive forward with negative Y (forward)
+                              * MaxSpeed
+                              * (driverController.start().getAsBoolean()
+                                  ? 0.5
+                                  : 1), // Drive forward with negative Y (forward)
```

Comment:
Would be better to read `driverController.start()` once inside the lambda, store the result (0.5 or 1.0) in a local variable and then use that variable in each of these as a multiplier for the max speed or max angular rate

Reply:
FYI, the comment here is more about seeing the exact same block of code 3 times, and less about any computation / memory overhead from making the exact same set of calls 3 times.